library/hawkular_alerts_group_trigger.py

```diff
@@ -184,23 +204,29 @@ def conditions_update_required(self, trigger_id, desired_conditions):
                     return True
         return False
 
-    def required_updates(self, trigger, name, severity, enabled):
+    def required_updates(self, trigger, name, event_text, severity, auto_resolve, tags, enabled):
         """ Checks whether an update is required for the group trigger
 
             Returns:
-                Empty Hash (None) - If the name, severity and enabled status passed
+                Empty Hash (None) - If the name, event_text, severity, auto_resolve, tags and enabled status passed
```

Comment:
what does "Empty Hash (None)" mean?  `{} != None`.  Just say "Empty Hash".

I'd also stop listing here (twice) all the attributes compared, just say that all values match.